cli/src/file_utils.rs

```diff
@@ -42,9 +42,15 @@ pub fn get_files(directory: &str, paths_to_ignore: &[String]) -> Result<Vec<Path
 
         // check if the path should be ignored by a glob or not.
         for path_to_ignore in paths_to_ignore {
-            // we build the expanded glob to make sure it will match with the full path.
-            let expanded_glob = format!("{}/{}", directory, path_to_ignore);
-            if glob_match(expanded_glob.as_str(), entry.display().to_string().as_str()) {
+            let path_buf = entry.to_path_buf();
+
+            let relative_path = path_buf
+                .as_path()
+                .strip_prefix(directory)
+                .unwrap()
+                .to_str()
+                .expect("should get the path");
```

Comment:
What about this? Removing the unwraps and using a result instead to avoid panics.
```suggestion
            let relative_path = path_buf
                .strip_prefix(directory)
                .ok()
                .and_then(|p| p.to_str())
                .ok_or_else(|| anyhow::Error::msg("should get the path"))?;
```